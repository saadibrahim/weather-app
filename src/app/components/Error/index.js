import React from "react";
import PropTypes from "prop-types";
import { Typography } from "@material-ui/core";

function Error({ message }) {
  return (
    <Typography color="textSecondary" gutterBottom>
      {message}
    </Typography>
  );
}

Error.propTypes = {
  message: PropTypes.string.isRequired,
};

export default Error;
