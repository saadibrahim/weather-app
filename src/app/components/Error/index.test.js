import React from "react";
import renderer from "react-test-renderer";

import Error from "./index";

test("renders the Error using Snapshots", () => {
  const component = renderer.create(<Error message="test" />);
  expect(component).toMatchSnapshot();
});
