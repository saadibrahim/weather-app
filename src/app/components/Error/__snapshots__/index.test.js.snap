// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Error using Snapshots 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
>
  test
</p>
`;
